pkg/rtc/participant.go

```diff
@@ -1355,15 +1355,29 @@ func (p *ParticipantImpl) onSubscribedMaxQualityChange(trackID livekit.TrackID,
 		SubscribedQualities: subscribedQualities[0].Qualities, // for compatible with old client
 		SubscribedCodecs:    subscribedQualities,
 	}
+	// get track's layer dimensions
+	layers := make(map[livekit.VideoQuality]livekit.VideoLayer)
+	trackInfo := p.UpTrackManager.GetPublishedTrack(trackID)
+	if trackInfo != nil {
```

Comment:
rename to `track`? This is actually an interface. As we have `TrackInfo` as a protocol structure, would be good not to confuse here.